Gemfile.lock

```diff
@@ -185,17 +185,15 @@ GEM
       gli (~> 2.9)
       i18n (~> 0.7, >= 0.7.0)
       rubyzip (~> 1.0)
-    cucumber (2.4.0)
+    cucumber (2.0.2)
       builder (>= 2.1.2)
-      cucumber-core (~> 1.5.0)
-      cucumber-wire (~> 0.0.1)
+      cucumber-core (~> 1.2.0)
       diff-lcs (>= 1.1.3)
-      gherkin (~> 4.0)
+      gherkin (~> 2.12)
```

Comment:
Wow - that's a significant gherkin downgrade.  I assume this just puts us back where we started though :)